src/View/View.php

```diff
@@ -85,8 +85,14 @@ public function render(): string
         $contents = view($this->templateViewName(), $cascade);
 
         if ($this->shouldUseLayout()) {
+            if (Str::endsWith($this->layoutViewPath(), '.blade.php')) {
+                $renderedContent = $contents->render();
+            } else {
+                $renderedContent = $contents->withoutExtractions()->render();
+            }
```

Comment:
I feel like this should be swapped and say "if it's `antlers.html/php` template"

Otherwise I think we'll have the same problem with other view languages like plain html, twig, etc.

Reply:
I've replaced the hard coded `.antlers.html` to now look at the `EXTENSIONS` defined on `Statamic\View\Antlers\Engine` so they don't have to be kept in sync in the future.

The default behavior of all non-Antlers extensions is to now exclude the `withoutExtractions()` method call.

Reply:
Even better! Forgot about that array.